### ISSUE TITLE
Mastodon profile does not update

### ISSUE THREAD

**Issue by bpwhite** [snarfed/bridgy-fed#1612]
I changed my default username a few weeks ago from crvscience.bsky.social to a custom domain, crvscience.substack.com and my Mastodon bridged account (@crvscience.substack.com@bsky.brid.gy) appears to have stopped updating as of 4 days ago. My fediverse bridgy page is still clearly updating: https://fed.brid.gy/bsky/crvscience.substack.com

**Comment by Tamschi** [comment]
Which domain are you checking this from?

ActivityPub instances don't receive posts directly unless there's at least one local follower. (This is inherent to ActivityPub.)

**Comment by snarfed** [comment]
Oh also, sadly we don't fully support Bluesky handle changes yet, partially (imo largely) because the fediverse itself is bad at supporting username changes: https://github.com/snarfed/bridgy-fed/issues/1074

**Comment by bpwhite** [comment]
So is there a way to force a full delete/reset?

**Comment by bpwhite** [comment]
> Yes, I can do that if you're ok with losing all of your past bridged posts, just let me know.

I blocked the FediBridge account on Bluesky, if you do the reset and then I unblock will it put in back in place with a new account? (Assuming you do the reset for new username). I'm willing to be a tester on this one 🙂 

Presumably it wipes out the current somewhat broken Mastadon account.

**Comment by snarfed** [comment]
Oh sorry, I forgot that this was due to a Bluesky handle change. I don't yet have a manual process for "fixing" those accounts. Sorry for the bad news. Feel free to follow #1074, hopefully eventually!

**Comment by snarfed** [comment]
Duplicate of #1074
